### ISSUE TITLE
Settings value of NONE in test case table for [Setup]

### ISSUE THREAD

**Issue by Daniel-10** [nokia/RED#312]
Having a settings value of None in a test case table [Setup] entry seems to give an issue

```
*** Settings ***
Documentation    NONE
  

*** Test Cases ***

test
    [Documentation]    NONE
    [Tags]    NONE
    [Setup]    NONE
    [Template]    NONE
    Comment    "hello"
```

[Setup]    NONE gives an error message of   " Unknown keyword 'NONE' " .  It does run in Robot but I'm not sure if I entered something incorrectly 

![Capture](https://user-images.githubusercontent.com/51714470/59288489-e983c280-8c41-11e9-9d08-de404da84b82.PNG)

**Comment by michalanglart** [comment]
> [Setup] NONE gives an error message of " Unknown keyword 'NONE' " . It does run in Robot but I'm not sure if I entered something incorrectly

Somehow RED never supported that although Robot user guide tells that everything is correct. Thanks for spotting

**Comment by Daniel-10** [comment]
No problem.  Thanks for making and maintaining this awesome tool.  It looks like this is the only tool that allows debug in both robot and python.  I think my colleagues will start using too when they see how useful the  features are!

One more question, in the paths tab for RED.xml would it be possible for it to support the wildcards 'folder/*' to include the subdirectories instead of having to write out the paths for each subdirectory?  No big deal if not, because for this I just used glob to generate all the paths and copy and pasted them into the paths in RED.xml.

**Comment by michalanglart** [comment]
> One more question, in the paths tab for RED.xml would it be possible for it to support the wildcards 'folder/*' to include the subdirectories instead of having to write out the paths for each subdirectory? No big deal if not, because for this I just used glob to generate all the paths and copy and pasted them into the paths in RED.xml.

Please report in separate issue. It doesn't look like something of high priority, so don't expect it in nearest releases.

**Comment by michalanglart** [comment]
Fixed in 0.9.0. Additionally content assistant proposes NONE in settings where it makes sense
